stories/core/Table.stories.tsx

```diff
@@ -1194,3 +1194,6 @@ Full.args = {
   isSortable: true,
   emptyFilteredTableContent: 'No results found. Clear or try another filter.',
 };
+
+export const Condensed: Story<TableProps> = Basic.bind({});
+Condensed.args = { density: 'extra-condensed' };
```

Comment:
Story name should probably match the prop.

If you rename the story, don't forget to remove the old test image.

Also, can you add `density: default` to the top level `args`?

Reply:
So the story should be named Extra-Condensed?

Reply:
The variable will be `ExtraCondensed`, and it will look like `Extra Condensed` in storybook.

Still not sure about the name though. Maybe `Density` would be better?